apps/passport-server/src/database/queries/pretixEventInfo.ts

```diff
@@ -64,3 +64,19 @@ export async function softDeletePretixEventsInfo(
     [id]
   );
 }
+
+export async function fetchPretixEventInfoByName(
```

Comment:
is the event name unique? if not, this would be problematic. the best index for an event is either `eventConfigID`, or the `organizer_url` + `event_id` from pretix.

Reply:
https://github.com/proofcarryingdata/zupass/issues/662